data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,851 @@ mission "Coalition License Patch"
 	on offer
 		set "license: Coalition"
 		fail
+
+
+
+mission "Kimek Student 1"
+	minor
+	name "Transport Chikee"
+	description "Chikee, a young Kimek, is studying to become a Heliarch. Transport him to <destination>, where he was offered a job, so that he can make a living while studying on his own."
+	passengers 1
+	to offer
+		has "Coalition: First Contact: done"
+		random < 90
+	source "Fourth Shadow"
+	destination "Second Rose"
+	on offer
+		conversation
+			`At any given time there are thousands of Kimek here in the spaceport, either mingling among themselves during a break from work or negotiating travel prices with starship captains. You imagine that if you weren't a human and clearly stood out, you too would be constantly getting offers to travel to another world, as many civilian captains here wish to fill a last bunk or two they have vacant on their ship before departing.`
+			`	You manage to find a restaurant here where you spot some food you've already learned you can eat, so you take a seat and order something.`
```

Comment:
dunno, this may well be the last mission a player ever comes across in Coalition space, and by that point if they have been only in Coalition space doing missions and taking jobs for some ingame months it'd be odd to have this one say "you finally decide to try the food here"